core/src/test/java/org/elasticsearch/action/support/IndicesOptionsTests.java

```diff
@@ -60,29 +60,82 @@ public void testSerialization() throws Exception {
     }
 
     public void testFromOptions() {
-        int iterations = randomIntBetween(5, 20);
-        for (int i = 0; i < iterations; i++) {
```

Comment:
No, I'm ok with it as is. I was just wondering if there was alternative motive behind this change other than it isn't really needed.